undead-discord.css

```diff
@@ -36,19 +36,24 @@
 /* The following 3 blocks color the keybind-shortcut modal and buttons */
 .ddr-arrows { display: none; }
 
-.keybind-shortcut > span {
-    color: #4F545C !important;
+.keybind-shortcut > span,
+.keybind-shortcut > span > svg > g {
+    color: var(--fg-color);
     background-color: var(--secondary-bg-color) !important;
     border: none;
     box-shadow: none;
     height: 21px;
+    padding: 5px 6px;
+    fill: var(--fg-color);
```

Comment:
Same as the other fill, I doubt all these variables need to be set for both CSS elements.